README.md

```diff
@@ -159,38 +171,224 @@ function App() {
 export default App;
 ```
 
-You can pass Bridge options as a third parameter as well:
+`StoryblokComponent` renders the route components dynamically, using the list of components loaded during the initialization inside the `storyblokInit` function. 
+
+This is how you can pass the Bridge options as a third parameter to `useStoryblok`:
 
 ```js
-useStoryblok(story.id, (story) => (state.story = story), {
+useStoryblok(story.id, {version: 'draft'}, {
   resolveRelations: ["Article.author"],
 });
 ```
 
-#### 3. Link your components to Storyblok Visual Editor
+**Check out our React Boilerplate [here](https://github.com/storyblok/storyblok-react-boilerplate), or read on how to add Storyblok to React in 5 mins [here](https://www.storyblok.com/tp/headless-cms-react)**
+You can also take a look at the [React Playground](https://github.com/arorachakit/storyblok-react/tree/main/playground) in this repo.
 
-For every component you've defined in your Storyblok space, call the `storyblokEditable` function with the blok content:
+
+## Next.js using App Router - Full React Server Components
```

Comment:
@arorachakit as discussed on Slack, should we swap this section with "Next.js using App Router - Live Editing support"?

Reply:
@fgiuliani I've swap the sections and adjusted the content according to it. Please take a look.